doc/make_qbk.py

```diff
@@ -198,7 +198,7 @@ def cs_to_quickbook(section):
 class_to_quickbook2("de9im_1_1static__mask", "de9im_static_mask")
 
 os.chdir("index")
-exec(open("make_qbk.py").read())
+execfile("make_qbk.py")
```

Comment:
Why have you reverted it back?

Reply:
I was using Python3 so I changed that and I revert it to make it as same as before.